bin/get-cutouts.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+
+#salloc -N 4 -C cpu -A desi -L cfs -t 04:00:00 --qos interactive --image=dstndstn/viewer-cutouts 
+#srun -n 4 -c 128 shifter /global/homes/i/ioannis/code/desihub/fastspecfit/bin/get-cutouts.sh fuji 128 > /global/cfs/cdirs/desi/spectro/fastspecfit/fuji/v2.0/logs/cutouts-fuji-01.log 2>&1 &
+
+codedir=/global/homes/i/ioannis/code/desihub
+mpiscript=$codedir/fastspecfit/bin/get-cutouts
+
+outdir_data=/global/cfs/cdirs/desi/spectro/fastspecfit
+outdir_html=/global/cfs/cdirs/desi/users/ioannis/fastspecfit
```

Comment:
It's great to include the actual slurm scripts used as a starting point for your future self, and I'm sympathetic to shortcuts to get work done, but it's usually a red flag when a code repo gets a hardcoded path to a specific user's directory at a specific computing center.  Maybe this is sufficient/pragmatic to get 2.0 out the door, but consider whether you can have a more flexible script/directory pattern in the future to be able to re-use these as-in for future prods (or by future people other than yourself).  Same comments apply to get-cutouts.slurm .